packages/@o3r/components/builders/component-extractor/index.js

```diff
@@ -8,5 +8,6 @@ It should not be part of the package.
 const {resolve} = require('node:path');
 
 require('ts-node').register({ project: resolve(__dirname, '..', '..', 'tsconfig.builders.json') });
+require('ts-node').register = () => {};
```

Comment:
Why do we have to add that ?

Reply:
Workaround to the multiwatcher builder case that was calling register several times for each child builder.
This is a way (did not really found another :S) to have an unique call to the register function